### PR TITLE
Jetpack: 11.1-a.1 changelog editorial

### PR DESCRIPTION
**Pull request by samiff**
#### Changes proposed in this Pull Request:

Jetpack: 11.1-a.1 changelog editorial

#### Jetpack product discussion

n/a

#### Does this pull request change what data or activity we track or use?

No.

#### Testing instructions:

Proofread.

### CODE REVIEW

File: projects/plugins/jetpack/readme.txt [hunk 1]
```diff
@@ -242,39 +242,22 @@ Jetpack Backup can do a full website migration to a new host, migrate theme file
 4. Promote your newest posts, pages, and products across your social media channels.
 
 == Changelog ==
-### 11.0-beta - 2022-05-31
+### 11.1-a.1 - 2022-06-08
+#### Major Enhancements
+- Blocks: introduce a new Recipe block to display an easy to read recipe anywhere on your site.
```
Comment (jeherve) [inline projects/plugins/jetpack/readme.txt:247]: I wonder if we should leave this out of the "major" section for now, since the block is still in Beta?

**Review by jeherve — COMMENTED** [review]
Looks good. I just had one question.

File: projects/plugins/jetpack/readme.txt (hunk 1 above)
Comment (samiff) [reply projects/plugins/jetpack/readme.txt:247]: Ah wasn't aware, just drop it out for now?

File: projects/plugins/jetpack/CHANGELOG.md [hunk 2]
```diff
@@ -7,35 +7,30 @@
...
 - Reorder JS imports for `import/order` eslint rule. [#24601]
-- Updated package dependencies. [#24510]
-- Updated package dependencies. [#24576]
-- Updated package dependencies. [#24596]
-- VideoPress: add "allow='clipboard-write'" in iframe for embed in order to fix the Clipboard API on Chrome [#24616]
-- VideoPress: make sure "false" will be casted as false for useaveragecolor [#24615]
+- Updated package dependencies. [#24510, #24576, #24596]
+- VideoPress: add "allow='clipboard-write'" in iframe for embed in order to fix the Clipboard API on Chrome. [#24616]
+- VideoPress: make sure "false" will be casted as false for useaveragecolor. [#24615]
+- WoA: add check to prevent duplicate Custom CSS menu item on plan-less sites. [#24631]
```
Comment (coder-karen) [inline projects/plugins/jetpack/CHANGELOG.md:32]: Not under bug fixes? (The WoA Custom CSS menu item fix)

Comment (samiff) [reply projects/plugins/jetpack/CHANGELOG.md:31]: I tend to leave more WPcom centric things under "Other".

File: projects/plugins/jetpack/readme.txt (hunk 1 above)
Comment (samiff) [reply projects/plugins/jetpack/readme.txt]: How does https://github.com/Automattic/jetpack/pull/24674/commits/68d2f4d59e2a9bb8b200661a390876d81dbcfc6b look?

File: projects/plugins/jetpack/CHANGELOG.md (hunk 2 above)
Comment (coder-karen) [reply projects/plugins/jetpack/CHANGELOG.md:31]: 👍

File: projects/plugins/jetpack/readme.txt (hunk 1 above)
Comment (jeherve) [reply projects/plugins/jetpack/readme.txt]: That works for me. 👍

**Pull request merged**